### PR TITLE
Described adding _redirect file for deploying to Netlify

### PR DESCRIPTION
**Pull request by fozy81**
Here's a draft to fix issue #1645 using _redirect.

### CODE REVIEW

File: guides/release/getting-started/quick-start.md [hunk 1]
```diff
@@ -425,14 +425,22 @@ First you need to [sign up for a Netlify account](https://app.netlify.com/signup
...
 ![deploying to Netlify](/images/quick-guide/netlify/create-netlify-account.png)
 
-This section will walk you through two ways to deploy your ember application to production using the Netlify platform:
+The next step is to let the web app server know how to handle URLs.
+For Netlify, you can create a file in your `ember-quickstart` folder called
+`_redirects`. Add `/* /index.html 200` to the first line and save the file. 
+This will let the server know to redirect all pages to `index.html` file. 
+Once redirected, Ember.js app itself will generate the matching html for URLs such as `/scientists`.
+Or, you can use tools created by the community to help make deploying easier,
+such as [ember-cli-netlify](https://github.com/shipshapecode/ember-cli-netlify).
```
Comment (ijlee2) [inline guides/release/getting-started/quick-start.md:434]:
```suggestion
Alternatively, you can use an addon created by the community, such as [ember-cli-netlify](https://github.com/shipshapecode/ember-cli-netlify), to handle URLs.
```

File: guides/release/getting-started/quick-start.md [hunk 2]
```diff
@@ -425,14 +425,22 @@ First you need to [sign up for a Netlify account](https://app.netlify.com/signup
 
 ![deploying to Netlify](/images/quick-guide/netlify/create-netlify-account.png)
 
-This section will walk you through two ways to deploy your ember application to production using the Netlify platform:
+The next step is to let the web app server know how to handle URLs.
+For Netlify, you can create a file in your `ember-quickstart` folder called
```
Comment (ijlee2) [inline guides/release/getting-started/quick-start.md:429]:
```suggestion
The next step is to let the web app server know how to handle URLs. There are 2 ways to do so.

One, you can create a file in your `ember-quickstart` folder called
```

File: guides/release/getting-started/quick-start.md [hunk 3]
```diff
@@ -425,14 +425,22 @@ First you need to [sign up for a Netlify account](https://app.netlify.com/signup
...
+
+Now you are ready to deploy your app to production on Netlify platform. There are two ways to do this:
 
 1. Deploying to Netlify using drag and drop
 2. Deploying to Netlify using Git (specifically GitHub)
 
 **Deploying to Netlify using drag and drop**
 
-This assumes you have already created the `dist/` directory by running this command
+You may need to re-create your `dist/` directory to include changes made to `_redirects` file by running this command
```
Comment (ijlee2) [inline guides/release/getting-started/quick-start.md:443]: The trailing slash looked a little weird to me. I'd have expected `/dist` or just `dist`.

```suggestion
You may need to re-create your `dist` directory to include changes made to `_redirects` file by running this command
```

**Review by ijlee2 — APPROVED** [review]
I left a couple of suggestions that I think may help improve your paragraph. Can you have a look? Much thanks for helping out with closing the issue.

**Comment by fozy81** [comment]
> I left a couple of suggestions that I think may help improve your paragraph. Can you have a look? Much thanks for helping out with closing the issue.

Thanks @ijlee2, I've merged your suggestions.

**Review by ijlee2 — APPROVED** [review]
🥳

**Comment by ijlee2** [comment]
@fozy81 Can you have a look at whether linting and testing are failing locally? If so, can you push a commit that can fix the errors?

**Comment by fozy81** [comment]
@ijlee2 - I made a small change - both mentions of `dist` folder are now referred to as `dist`. All tests passing :fireworks: !

**Pull request merged**